src/test/java/eu/xlime/kafka/rdf/TVOCRToMongoITCase.java

```diff
@@ -35,11 +35,14 @@ public void testProcessData() throws Exception {
 		assertTrue(ds.isPresent());
 		
 		
-		boolean result = testObj.processDataset(mm, ds.get());
-		assertTrue(result);
+		testObj.processDataset(mm, ds.get());
+		testObj.processDataset(mm, ds.get());
+		testObj.processDataset(mm, ds.get());
 		String summary = testObj.generateSummary();
 		System.out.println("summary: " + summary);
 		assertNotNull(summary);
+		Summary sum = testObj.generateObjectSummary();
+		System.out.println(sum.toString());

```

Comment:
Now that we have a Summary object, improve the test by verifying specific values. E.g. we can add things like 
  `assertEquals(2, sum.getProcessed())`
  `assertEquals(2, sum.getAnnotations.get("OCRAnnotation_stored"))`